xbmc/dialogs/GUIDialogSelect.h

```diff
@@ -41,6 +41,7 @@ class CGUIDialogSelect : public CGUIDialogBoxBase
   void SetSelected(const std::vector<std::string> &selectedLabels);
   void SetUseDetails(bool useDetails);
   void SetMultiSelection(bool multiSelection);
+  void SetButtonFocus();
```

Comment:
```suggestion
  void SetButtonFocus(bool buttonFocus);
```
so it's consistent with the rest of the setters.